backend/src/main/java/com/team28/sidedish/service/CategoryService.java

```diff
@@ -1,10 +1,23 @@
 package com.team28.sidedish.service;
 
 import com.team28.sidedish.controller.dto.CategoryResponse;
+import com.team28.sidedish.repository.CategoryRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.context.annotation.Primary;
+import org.springframework.stereotype.Service;
 
 import java.util.List;
+import java.util.stream.Collectors;
 
-public interface CategoryService {
+@Service
+@RequiredArgsConstructor
+public class CategoryService {
 
-    List<CategoryResponse> findAll();
+    private final CategoryRepository categoryRepository;
+
+    public List<CategoryResponse> findAll() {
+        return categoryRepository.findAll().stream()
+                .map(categoryEntity -> CategoryResponse.of(categoryEntity.getId(), categoryEntity.getName()))
+                .collect(Collectors.toUnmodifiableList());
```

Comment:
이 부분을 아래처럼 메소드로 추출하는 형식에 대해서는 어떻게 생각하시나요?

```suggestion
        return convertToCatetoryResponses(categoryRepository.findAll());
```